apps/haproxy/haproxy.cfg

```diff
@@ -33,14 +33,18 @@ acl host_prometheus_backend hdr_dom(host) -i prometheus.ly.fish.local
 use_backend prometheus_backend if host_prometheus_backend
 
 frontend tcp_5432_in
-mode tcp
-bind *:5432
-default_backend postgres_backend
+  bind *:5432
+  mode tcp
+  log global
```

Comment:
Do we need to change log settings? If I'm not mistaken this change isn't really useful as we don't have anything log-related configured in the `global` section at the top of the file.

Reply:
Correct, this is mostly a leftover, I'll remove it